docker/base/cleanup.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+#
+# Cleanup script for Firefly app
+
+# /usr/bin/find if testing locally
+find_cmd="/bin/find"
+
+function doCleanup() {
+    workarea="${1}"
+
+    # find cleanup logs less than 23 hours old
+    recent_cleanup=`${find_cmd} "${workarea}/cleanup_logs" -type f -mmin -$((60*23))`
+
+    if [[ -z "${recent_cleanup}" ]] ; then
+
+        timestamp=$(date +20%y%m%dT%H%M%S)
+        log_file="${workarea}/cleanup_logs/cleanup.${timestamp}.log"
+
+        echo "Cleaning up old work files in ${workarea}" > "${log_file}" 2>&1
+        ${find_cmd} "${workarea}/stage" -type f -mtime +7 -exec /bin/rm '{}' \+ -print >> "${log_file}" 2>&1
+        # master catalogs should be cleared more often than other files
+        ${find_cmd} "${workarea}/perm_files" -name 'master*' -mtime +1 -exec /bin/rm '{}' \+ -print >> "${log_file}" 2>&1
+        ${find_cmd} "${workarea}/perm_files" -type f -mtime +7 -exec /bin/rm '{}' \+ -print >> "${log_file}" 2>&1
+        ${find_cmd} "${workarea}/temp_files" -type f -mtime +1 -exec /bin/rm '{}' \+ -print >> "${log_file}" 2>&1
+        ${find_cmd} "${workarea}/visualize" -type f -mtime +2 -exec /bin/rm '{}' \+ -print >> "${log_file}" 2>&1
```

Comment:
visualize is really two dirs. `visualize/users` should be one day.  It is high volume and very temporary. `visualize/fits-cache` is more similar to perm_files.  It could be 7 days.